packages/golden-layout/src/config/ItemConfig.ts

```diff
@@ -86,6 +86,11 @@ export interface ReactComponentConfig extends ItemConfig {
    * Properties that will be passed to the component and accessible using this.props.
    */
   props?: any;
+
+  /**
+   * The stack the component is in.
+   */
+  parentStackId?: string | string[];
```

Comment:
I don't think we want this to be part of the `ItemConfig` - this is something that changes as the component is moved around, and should be derived from it's position in the layout, not specified on the config itself.
Similar reasoning for why `ItemConfig` doesn't have a link to it's parent.